.github/CODEOWNERS

```diff
@@ -2,4 +2,3 @@
 # Each line is a file pattern followed by one or more owners.
 
 *                                    @MetaMask/mobile-devs
```

Comment:
Are we confident that this is the right format? Are we gonna accidentally lock ourselves out?

Reply:
I guess it matches https://github.com/MetaMask/metamask-mobile/blob/24b4b7eb389caea486e7c5030961e888866a81e7/.github/CODEOWNERS#L3

Reply:
I'm just seeing this weird warning:
![Screenshot 2023-10-05 at 12 13 06 PM](https://github.com/MetaMask/mobile-provider/assets/34557516/b63f8648-9a56-4209-a060-556328993714)